lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb

```diff
@@ -7,6 +7,8 @@ def initialize(aem, obj, inputs)
       @workspace = obj.workspace
       @inputs    = inputs
       @aem       = aem
+      @api_token = Api::UserTokenService.new.generate_token(@workspace.ae_user.userid, 'api')
+      @api_url   = MiqRegion.my_region.remote_ws_url
```

Comment:
I would prefer these not be in the initialize method and instead use memoization methods:

```ruby
  def api_token
    @api_token ||= Api::UserTokenService.new.generate_token(@workspace.ae_user.userid, 'api')
  end

  def api_url
    @api_url ||= MiqRegion.my_region.remote_ws_url
  end
```